### PR TITLE
Pull charts from ember-simple-charts

### PR DESCRIPTION
**Pull request by jrjohnson**
This is our extracted addon for chart presentation.
- [x] Click test to ensure our charts still work

### CODE REVIEW

**Comment by jrjohnson** [comment]
Deploy preview ready!

Built with commit c20c4c41a4c9748ae7ec787fe083d0e37c66e148

https://deploy-preview-3049--ilios-frontend.netlify.com

File: app/styles/newcomponents/ilios-chart-tooltip.scss [hunk 1]
```diff
@@ -1,37 +0,0 @@
-.ilios-chart-tooltip {
```
Comment (gabycampagna) [inline app/styles/newcomponents/ilios-chart-tooltip.scss]: Tooltip styling is missing when testing.  Content does render.
screen shot 2017-06-23 at 7 44 02 am

Comment (jrjohnson) [inline app/styles/newcomponents/ilios-chart-tooltip.scss]: Good catch, I've pushed a new version of simple-charts and required it in this branch.

**Comment by jrjohnson** [comment]
@gabycampagna this is ready for review again.

**Pull request merged**